haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights"
+COPYRIGHT="2017 Adrien Destugues"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f658b45894fe5d2349be4e06c3a0bce359985e27"
+SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9271f3778218bb63430da00d00b099c5d3e9d83b21d6c132533069b237199663"
+SOURCE_DIR="neonlights-$srcGitRev"
```

Comment:
No. We should never have to put git commands inside a recipe. That would be weird.
By "git mv" I meant typing "git mv old_name new_name" on the command line, in the directory where you wish to rename file "old_name" to "new_name".
This is because the file you have included in your commit needs to be renamed. If you rename it with "mv" (instead of "git mv") then you would have to add it with the new name. Using "git mv" allows to make the change in a single operation. git even does the mv for you, at the same time.
You can try
```
man git-mv
```
on the command line, btw.